### PR TITLE
LPS-162822 Test Integration and Simplify validation

### PR DESCRIPTION
**Pull request by murilostodolni**
Test Integration to: https://issues.liferay.com/browse/LPS-162822

Simplify validation to: https://issues.liferay.com/browse/LPS-171196

### CODE REVIEW

**Comment by murilostodolni** [comment]
Waiting for https://github.com/brianchandotcom/liferay-portal/pull/127944 to be merged.

**Comment by naraandrade** [comment]
> Waiting for [brianchandotcom#127944](https://github.com/brianchandotcom/liferay-portal/pull/127944) to be merged.

The PR was merged, @murilostodolni you can follow with this one.

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java [hunk 1]
```diff
@@ -319,6 +319,8 @@ public void testUpdateDefaultObjectStateFlow() throws Exception {
 
 	@Test
 	public void testUpdateObjectStateTransitions() throws Exception {
+		_step4ListTypeEntry = _addListTypeEntry("step4");
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:322]: This code could be in the setUp() method?

**Comment by murilostodolni** [comment]
ci:test:relevant

**Review by marcelabc — COMMENTED** [review]
Hi @murilostodolni 
I finished my review and I have one question :smile:

**Comment by murilostodolni** [comment]
ci:stop

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java (hunk 1 above)
Comment (murilostodolni) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java]: Perfect, i agree.

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by murilostodolni** [comment]
Forwarding manually: https://github.com/brianchandotcom/liferay-portal/pull/128185

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**